Modules/DevTools/TestFramework/TestRunner/src/ALTestTool.Page.al

```diff
@@ -303,7 +304,13 @@ page 130451 "AL Test Tool"
                     var
                         TestMethodLine: Record "Test Method Line";
                         TestSuiteMgt: Codeunit "Test Suite Mgt.";
+                        DeleteQuestion: Label 'Are you sure you want to Delete the Selected Lines?';
```

Comment:
Minor - move to global variables since it will be translated.

Reply:
Do you mean that if all these new Labels are defined as Local they won't be translated?

Do you want me to move all of them, including the DialogContent?

Reply:
Yes all of them -  They may be translated, however I'm not 100% certain if the internal tooling will pick it up. We add these as globals in all code, thus it is good to align.

Reply:
OK, I will make this change... thanks...